openhtf/io/output/mfg_inspector.py

```diff
@@ -129,14 +128,13 @@ def _EnsureUniqueParameterName(name, used_parameter_names):
 
 def _AttachJson(record, testrun):
   """Attach a copy of the JSON-ified record as an info parameter.
-
   Save a copy of the JSON-ified record in an attachment so we can access
   un-mangled fields later if we want.  Remove attachments since those get
   copied over and can potentially be quite large.
   """
   record_dict = data.ConvertToBaseTypes(record, ignore_keys=('attachments',))
   record_json = json_factory.OutputToJSON(
-      sort_keys=True, indent=2).encode(record_dict)
```

Comment:
oh, where you have `ignore_keys='attachments'`, you need to instead have `ignore_keys=('attachments',)`.. and i'm glad you hit that, because it makes me feel good about having he foresight to put that assertion there lol... had it not been there, you would have accidentally ignored the keys 'a', 't', 'c', 'h', 'm', 'e', 'n', and 's', because it would have interpreted 'attachments' as an iterable of individual string keys.